client/src/components/AddNewProfile.js

```diff
@@ -54,6 +59,19 @@ const AddNewProfile = () => {
     { key: "inactive", value: "inactive", text: "Inactive" }
   ];
 
+  const groupsOptions = [
+    { key: 1, value: 1, text: "MIN Voices" },
+    { key: 2, value: 2, text: "Oasis Women Group" },
+    { key: 3, value: 3, text: "Men Group" },
+    { key: 4, value: 4, text: "Family Group" },
+    { key: 5, value: 5, text: "Gardening Group" },
+    { key: 6, value: 6, text: "Joyous Choir" },
+    { key: 7, value: 7, text: "ESOL" },
+    { key: 8, value: 8, text: "Volunteer" },
+    { key: 9, value: 9, text: "Knit for Unity" },
+    { key: 10, value: 10, text: "Echo Dance Project" }
+  ];
+
```

Comment:
Now that Hacer's group filtering code has been added, with an API to fetch the groups, let's use that here instead of a hard coded list of options.